### ISSUE TITLE
TaskDialog position

### ISSUE THREAD

**Issue by Trude** [eugener/oxbow#14]
The TaskDialog's location is currently set relative to the window which has keyboard focus at the time  setVisible is called. Consider the following scenario: A user initiates an action which takes some time before resulting in a TaskDialog. Between the time of initiating the action and the TaskDialog becoming visible, the user may start using another window in the application on another monitor, thus changing the keyboard focus window before the TaskDialog's setVisible is called. As a result, the TaskDialog appears on the wrong monitor. Perhaps the location should be set relative to its parents location, not the active window. It would be nice to have an option to do so.
